### PR TITLE
Updating bug-triage handbook re exceptions process

### LINKED ISSUE #1012: Updating bug-triage handbook re exceptions process
**Anything else we need to know?**: Updating bug-triage handbook re exceptions process

### PR DESCRIPTION
**Pull request by markyjackson-taulia**
Signed-off-by: markyjackson-taulia <marky.r.jackson@gmail.com>

Fixes #1012 

**Special notes for your reviewer**:
Bug-Triage handbook update

### CODE REVIEW

**Comment by justaugustus** [comment]
Thanks Marky!
/hold for my review
/assign

**Comment by markyjackson-taulia** [comment]
@justaugustus os this ok to merge?

**Comment by markyjackson-taulia** [comment]
image
😂😂😂

**Comment by markyjackson-taulia** [comment]
@justaugustus and I had a chat about this and it will remain on hold until a few other handbooks get updated and at that point, I will need to make some changes to how I have this handbook worded.

File: release-team/role-handbooks/bug-triage/README.md [hunk 1]
```diff
@@ -208,7 +208,9 @@ Your responsibility here is to actively watch for any new issues/PRs targeting t
 
 On the day of the Code Freeze, your responsibility is to try to help contributors to get the approval on their PRs and needed label. Check [How To Escalate](#how-to-escalate) part of the document for guide how to do this.
 
-Please note that code freeze is EOD Pacific Time (astronomical sun) on the day of.
+Please note that code freeze is EOD Pacific Time (astronomical sun) on the day of. No items are to be removed from the milestone until the next day. 
+
+The day after the code freeze has begun, the bug-triage team will wait to see if new exception requests are filed and approved (see [exception process](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md) for more details), it is at this time you will remove the remaining issues / PRs that are not declined into the next milestone. It is important to ensure that the release lead has been informed during the burn down meeting, following code freeze, that items are being removed so they may provide any additional feedback
```
Comment (justaugustus) [inline release-team/role-handbooks/bug-triage/README.md]:
```suggestion
The day after the Code Freeze has begun, the Bug Triage team will wait to see if new exception requests are filed and approved (see [exception process](/releases/EXCEPTIONS.md) for more details), it is at this time you will remove the remaining issues / PRs that are not approved into the next milestone. It is important to ensure that the Release Team Lead has been informed during the Burndown meetings following Code Freeze that items are being removed so they may provide any additional feedback.
```

**Comment by markyjackson-taulia** [comment]
@justaugustus Thank you kindly, changes have been made. Let me know if anything else is needed

**Comment by justaugustus** [comment]
Thanks @markyjackson-taulia!
/lgtm
/approve
/hold cancel

**Pull request merged**